content/en/cloudcraft/getting-started/create-your-first-cloudcraft-diagram.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Create your first live AWS diagram
+kind: guide
+---
+
+Cloudcraft allows you to easily import your AWS cloud environment as *live diagrams*. By reverse-engineering the architecture in your AWS account, Cloudcraft can autogenerate new diagrams or enhance existing ones, saving you hours or even days of work.
+
+<section class="alert alert-info">
+  <p>If you would like to get a full tour of Cloudcraft and all its features, you can <a href="https://www.cloudcraft.co/request-demo" title="Book an online demo of Cloudcraft">book an online demo with a member of our team</a>.</p>
+</section>
+
+## Your first live diagram
+
+Start by creating a new blueprint, where we will be importing your cloud architecture.
+
+<section class="alert alert-info">
+  <p>A blueprint contains your diagram, a budget, and all the documentation you attach to individual components and the diagram itself.</p>
+</section>
+
+First things first, you need to connect your AWS account. Go ahead and do that by following the help article below:
+
+- [Connect your AWS account with Cloudcraft][1]
+
+Connected? Perfect!
+
+Click on the **Live** tab, which is where you can choose the account you just added, scan AWS regions, generate automatic layouts, and view all resources in your AWS account.
+
+{{< img src="cloudcraft/getting-started/create-your-first-cloudcraft-diagram/live-tab.png" alt="Screenshot of a live AWS infrastructure diagram on Cloudcraft platform with design, live, and budget tabs visible. The Live tab is highlighted." responsive="true" style="width:100%;">}}
+
+If you only added one AWS account to Cloudcraft, it will be automatically selected for you, otherwise, select the account you want from the dropdown.
```

Comment:
```suggestion
If you only added one AWS account to Cloudcraft, it's automatically selected for you, otherwise, select the account you want from the dropdown.
```